src/interfaces/libpq/fe-protocol3.c

```diff
@@ -475,6 +476,17 @@ pqParseInput3(PGconn *conn)
 					 */
 					break;
 #ifndef FRONTEND
```

Comment:
if defined FRONTEND, will any warnings ?

Reply:
No, it's quite common for this style.